src/test/java/com/stripe/functional/CardTest.java

```diff
@@ -136,14 +137,23 @@ public void testList() throws IOException, StripeException {
     assertNotNull(card);
   }
 
+  /**
+   * TODO(spastorelli): stripe-mock returns a bank account object when calling "Delete Card".
+   *  i.e DELETE  /v1/customers/cus_1234/sources/card_123
+   *  So object is deserialized into a BankAccount and since we now return a Card for delete() as
+   *  a result the test fails with a
+   *  ClassCastException (Cannot cast com.stripe.model.BankAccount to com.stripe.model.Card).
```

Comment:
Right. It's probably going to be a while until stripe-mock handles this one correctly unfortunately, so I would simply do something similar to the list test just above and mock the fixture.

Since the JSON is so simple, you don't even need an external file, you could do something like this:
```java
     stubRequest(
         ApiResource.RequestMethod.DELETE,
         String.format("/v1/customers/%s/sources/%s", customer.getId(), card.getId()),
         null,
         Card.class,
         String.format("{\"id\": \"%s\", \"object\": \"card\", \"deleted\": true}", card.getId())
     );
```

(untested, but you get the idea)

Reply:
Thanks @ob-stripe. I've stubbed the request as suggested.